src/blueprint.ts

```diff
@@ -166,6 +179,7 @@ export default class Blueprint extends Command {
   public async generate(argv: any): Promise<void> {
     let data = this.locals(argv);
     let dest = process.cwd();
+    let renamedFiles = assign({}, Blueprint.renamedFiles, this.renamedFiles);
```

Comment:
This kind of configuration for blueprints is probably static only. Instance level variation would only result from different command line arguments. In other words, every blueprint has it's own class, so the `this.renamedFiles` here would always be empty, since you probably wouldn't have a command line flag to add a renaming pattern.

If you're looking for extendability (i.e. my blueprint could include whatever the "default" renames are, plus a few more), you could do something like what actions do for filters - you recursively walk the prototype chain starting with `Object.getPrototypeOf(this.constructor)`, collecting the `renamedFiles` for each prototype as you go.

Honestly, in this scenario, that might be unnecessary. If you want to implement it by all means, but I think it would also be fine just to go with `this.constructor.renamedFiles` for now, and if it becomes a pain point later, we can add the prototype walking.